app/schema.py

```diff
@@ -12,13 +12,42 @@ class Meta:
         model = User
 
 
+class FriendRequestType(DjangoObjectType):
+    class Meta:
+        model = FriendRequest
+
+
+class FriendType(DjangoObjectType):
+    class Meta:
+        model = Friend
+
+
 class Query(ObjectType):
     user = graphene.Field(UserType, contact=graphene.String())
+    friend_request = graphene.Field(FriendRequestType, contact_receiver=graphene.String())
 
     @login_required
     def resolve_user(self, info, **kwargs):
         return info.context.user
 
+    @login_required
+    def resolve_friend_request(parent, info, **kwargs):
+        contact_receiver = kwargs.get('contact_receiver')
+        try:
+            friend_request = FriendRequest.objects.get(from_user=info.context.user, to_user__username=contact_receiver)
+            return friend_request
+        except FriendRequest.DoesNotExist:
+            return None
+
+    @login_required
+    def resolve_friend(parent, info, **kwargs):
```

Comment:
What does this do?

Reply:
This returns the friend object of certain user and the request user. Now I am thinking of changing it to return all friends of request user. What do you think?